frontend/src/components/dashboard/create/govsg/GovsgPickTemplate.tsx

```diff
@@ -10,6 +11,8 @@ import {
 
 import { useParams } from 'react-router-dom'
 
+import { getLocalisedTemplateBody } from '../../../../utils/templateLocalisation'
```

Comment:
Adding `utils/*` to the `tsconfig.paths.json` file worked locally but not when running the integration tests (see previously failed pipelines).